components/constituency_lookup/ConstituencyLookup.tsx

```diff
@@ -252,13 +299,13 @@ const PostcodeLookup = () => {
       apiResponse.constituencies.length == 0
     ) {
       // User hasn't input anything or invalid postcode
-      setError("POSTCODE_INVALID");
+      setPostError("POSTCODE_INVALID");
       return;
     }
 
     //not selected constituency or address
     if (apiResponse.constituencies.length > 1 && !formState.constituencyIndex) {
-      //TODO: Set error for unclear constituency
+      setPostError("UNCLEAR_CONSTITUENCY");
```

Comment:
Has only become apparent now, but the `if` above isn't right. It triggers if the constituency index is 0 (ie. you've chosen the first element in the select box).

Think we need `constituencyIndex !== false` or similar so it doesn't catch the `0` case.